hc/api/views.py

```diff
@@ -11,17 +11,23 @@
 from hc.api.models import Check, Ping
 from hc.lib.badges import check_signature, get_badge_svg
 
+from hc.api.management.commands.sendalerts import Command
+
 
 @csrf_exempt
 @uuid_or_400
 @never_cache
 def ping(request, code):
     try:
         check = Check.objects.get(code=code)
+
     except Check.DoesNotExist:
         return HttpResponseBadRequest()
+    # get the last_ping to update the previous_ping
+    previous_ping_time = check.last_ping
```

Comment:
This variable naming is confusing considering you have defined `previous_ping` and `last_ping` separately. Variable names should be intuitive.

Reply:
I'll get a better name for `previous_ping`, one that's more intuitive

Reply:
I've renamed `previous_ping` to `ping_before_last_ping`